test/unit/browser_collection.spec.js

```diff
@@ -198,7 +198,9 @@ describe('BrowserCollection', () => {
 
     it('should compute exitCode', () => {
       var browsers = [new Browser(), new Browser()]
-      browsers.forEach(collection.add)
+      browsers.forEach((browser) => {
```

Comment:
Why?

Reply:
I can't recall off the top of my head - I think that the test was causing weird scoping issues. Obviously we can change it back, but I don't semantically agree with the way it's being called there. `forEach` in JavaScript is already syntactically strange enough when you use it with an inline function, and here we're implicitly having it drop arguments (as forEach actually calls the function with `currentValue, index, array` as parameters).

Reply:
Sure, I see your point